Jenkinsfile.integration

```diff
@@ -23,7 +23,6 @@ pipeline {
     parameters {
```

Comment:
- Why has the commit msg (for 42cdde8ecfd6f56cacfd65dd96261fdfe44a17c4) a cherry-pick line? Doesn't make sense imo

Reply:
Because it was cherry-picked from the older one? I'm not sure I get your point

Reply:
removed